### PR TITLE
Create responsive tablet layout for venue map

### PR DESCRIPTION
**Pull request by sweetstyle11**

### CODE REVIEW

File: app/views/pages/info_venue_map.html.erb [hunk 1]
```diff
@@ -5,28 +5,86 @@
...
+      <div class="teardrop">
+        <div class="teardrop__color">I</div>
+        <div class="teardrop__color">I</div>
+        <div class="teardrop__color">I</div>
+        <div class="teardrop__color">I</div>
+        <div class="teardrop__color">I</div>
+        <div class="teardrop__color">I</div>
+      </div>
+    </div>
+  </div>
```
Comment (sumeetjain) [inline app/views/pages/info_venue_map.html.erb]: Remove trailing whitespace

**Comment by sweetstyle11** [comment]
When creating tablet views. Certain elements are actually getting smaller when the screen breaks to the larger size. For example: a "sidebar" that takes up the full screen width on mobile, but only a fraction of the screen width on tablet.  This is causing issues with text size, and text overflowing past its container's background color.  What is the best practice to resolve this? Should I make all the text smaller, should I decrease text size in the media query, or I also discovered that some people recommend all text size should actually be a percentage based of a set pixel value?

**Comment by sumeetjain** [comment]
> decrease text size in the media query

That.

The approach of setting font-size using percentage is valid, but it's going to take a lot more effort right now.

File: app/views/pages/homepage_mobile.html.erb [hunk 2]
```diff
@@ -1,86 +1,87 @@
...
-        <a href="#" class="day_link day_link--mid">
-          <span href="#" class="day_link__day">Friday</span>
-          <span href="#" class="day_link__date">7</span>
-        </a> 
+      <div class="clear dates">
+        <div class="col-4">
+          <a href="#" class="day_link day_link--left day_link--selected">
+            <span href="#" class="day_link__day">Thursday</span>
+            <span href="#" class="day_link__date">6</span>
+          </a>
```
Comment (sumeetjain) [inline app/views/pages/homepage_mobile.html.erb]: Remove trailing whitespace

File: app/views/pages/info_venue_map.html.erb [hunk 3]
```diff
@@ -5,28 +5,96 @@
...
+<!-- Venues -->
 <div class="clear">
-  <div class="col-10 venues">
-    <div class="venues__actual venues__actual--selected">Bourbon Theater</div>
-    <div class="venues__actual">Duffy's Tavern</div>
-    <div class="venues__actual">Fat Toad</div>
-    <div class="venues__actual">Bourbon Theatre</div>
-    <div class="venues__actual">Duffy's Tavern</div>
-    <div class="venues__actual">Fat Toad</div>
+
```
Comment (sumeetjain) [inline app/views/pages/info_venue_map.html.erb]: Remove trailing whitespace

**Pull request closed without merging**